File: Common/Entities/RoadSituation.cs
using Common.BusinessObjects;

namespace Common.Entities
{
    public class RoadSituation
    {
        public int Id { get; set; }

        public int DrivingTestId { get; set; }

        public DrivingTest DrivingTest { get; set; }

        public bool Success { get; set; }

        public bool HasSigns { get; set; }

        public bool HasTrafficLight { get; set; }

        public bool HasVip { get; set; }
    }
}

File: AutoSchool/ClientApp/src/app/shared/entities/RoadSituation.ts
import { DrivingTest } from './DrivingTest';


export class RoadSituation {
    constructor(
        public id?: number,
        public drivingTestId?: number,
        public drivingTest?: DrivingTest,
        public success?: boolean,
        public hasSigns?: boolean,
        public hasTrafficLight?: boolean,
        public hasVip?: boolean
    ) { }
}
